content/applications/hr/recruitment.rst

```diff
@@ -0,0 +1,726 @@
+
+Email templates
+~~~~~~~~~~~~~~~
+
+To communicate with the applicant, Odoo has several pre-configured email templates that can be used.
+The pre-configured email templates and when to use them are as follows:
+
+- :guilabel:`Applicant: Acknowledgement`: This template is used to let the candidate know that their
+  application was received. This email is automatically sent out once the candidate is in the
+  :guilabel:`Initial Qualification` stage.
+- :guilabel:`Applicant: Interest`: This template is used to let the candidate know that they have
+  passed the :guilabel:`Initial Qualification` stage and they will be contacted to set up an
+  interview with the recruiter.
+- :guilabel:`Applicant: Not interested anymore`: This template is used when a candidate communicates
+  that they are no longer interested in the position, and thanks them for their time and
+  consideration.
+- :guilabel:`Applicant: Refuse`: This template is used when an applicant is no longer being
+  considered for the position.
+- :guilabel:`Applicant: Your Salary Package`: This template is used when offering a candidate a
+  position. This informs the candidate they can configure their salary package.
+
+For this stage, it is recommended to use the :guilabel:`Applicant: Interest` email template.
+
+.. note::
+   Email templates can be created, modified, and deleted to suit the needs of a business. For more
+   information on email templates, refer to the :doc:`../general/email_communication/email_template`
+   document.
+
+To send an email using the :guilabel:`Applicant: Interest` email template, click :guilabel:`Send
+message` in the chatter. A text box appears, as well as the candidate's email address. To use an
+email template, click the :guilabel:`⤢ (Full composer)` icon in the bottom right corner of the
+:guilabel:`Send Message` tab in the chatter.
+
+.. image:: recruitment/full-composer.png
+   :align: center
+   :alt: Send an email using a template by clicking the full composer icon in the bottom right of
+         the send message tab in the chatter.
+
+An email pop-up loads, with the :guilabel:`Recipients` and :guilabel:`Subject` pre-populated. The
+candidate's email address is entered on the :guilabel:`Recipients` line, and the :guilabel:`Subject`
+is `Re: (Subject / Application Name)`. The email body is empty by default.
+
+To use an email template, click the drop-down next to the :guilabel:`Load template` field in the
+lower right section of the window. There are several pre-configured email templates to chose from.
+Select the :guilabel:`Applicant: Interest` email template, and the template loads in the email body.
+This email template states that the applicant is shortlisted for either a phone call or an
+interview, and provides the contact information for the person who is listed as the
+:guilabel:`Interviewer` on the job position, explaining they will be contacted by that person.
```

Comment:
```suggestion

```
I think it would be best to keep this section generic to describe just the act of sending a message via a template, not a specific template.

Reply:
Good idea- they can read what each template says